stylesheets/all.scss

```diff
@@ -62,3 +62,16 @@ $screen-sm-min: 576px;
   justify-content: space-between;
   align-items: flex-end;
 }
+
+.cancel-icon {
+  width: 20px;
+  height: 20px;
+  top: 5px;
+  position: relative;
+  cursor: pointer;
+}
+
+#loading-spinner {
```

Comment:
The problem is not the class, it's the global file and at global level 😅 
We should have a discussion about styling and pick if we prefer going with css modules or css-in-js and stick with that. So we can keep this file for generic rules and override the govuk one.

Reply:
Since you've added your desired change, I've removed this https://github.com/LBHackney-IT/lbh-social-care/blob/573093a5f1a138866a7d99b60ca286078a7d548c/components/Spinner/Spinner.jsx#L8